hw3/01-characters.js

```diff
@@ -1,4 +1,24 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const section = document.querySelector('section');
 
-fetch(url);
+fetch(url)
+  .then((response) => response.json())
+  .then((data) => {
+    for (const character of data) {
+      const card = document.createElement('card');
+      const image = document.createElement('img');
+      image.src = character.imageUrl;
+      image.height = '200';
```

Comment:
I'd suggest using css to style these instead of doing it in js.  I think it will make it easier to read and it might also be a better workflow.

Reply:
In addition to that comment you can add class names in the JS so the CSS woul get applied. For example: element.**classList.add("mystyle");**